plugin/code_actions.py

```diff
@@ -312,6 +313,10 @@ def run_async() -> None:
                 selected = self.commands[index]
                 session = self.session_by_name(selected[0])
                 if session:
-                    session.run_code_action_async(selected[2], progress=True)
+                    session.run_code_action_async(selected[2], progress=True).then(self.handle_response_async)
 
             sublime.set_timeout_async(run_async)
+
+    def handle_response_async(self, response: Any) -> None:
+        if isinstance(response, Error):
+            sublime.error_message(str(response))
```

Comment:
Can we maybe include some information like server name at least? I think that depending on the error message that can seem a bit unclear.